### DISCUSSION TITLE
devrt scoring system: forcing local worker to run workload

### DISCUSSION THREAD

**Discussion by tlaw009** [hsr-project/general#18] [Q&A]
As of this moment, the central server seems to have the priority when it comes to running a workload. Is there a way to force a local worker to run a workload instead? It would be great if we can do rapid testing without the risk of clogging up the central server.

**Comment by yosuke** [comment]
I agree this is the current most wanted feature.
There is actually no needs to use the central worker if you already have a good GPU server connected.
I will queue this as a high-priority TODO item.
For the next robocup session, we are planning to power up the central GPU server which hopefully solves many problems.
